android/sdl_android/src/main/java/com/smartdevicelink/util/SdlAppInfo.java

```diff
@@ -181,6 +181,40 @@ public String toString() {
         }
         return vehicleMakesList;
     }
+
+    static boolean checkIfVehicleSupported(List<VehicleType> supportedVehicleList, VehicleType connectedVehicle){
+        if (null == supportedVehicleList || null == connectedVehicle){
```

Comment:
@kostyaBoss On line 186 I would change the ordering to be:
`if (supportedVehicleList == null || connectedVehicle == null)`